parser.go

```diff
@@ -98,14 +101,17 @@ func parseImports(impDefs []ImportDefinition, resolver Resolver, hooks ParserHoo
 		if err != nil {
 			return std, err
 		}
-		err = hooks.ParsedSTD(im.File, &tt)
+		err = hooks.ParsedSTD(imFilePath, &tt)
 		if err != nil {
 			return std, err
 		}
 
 		if len(tt.Imports) != 0 {
+			if isLocalPath(imFilePath) && filepath.IsAbs(imFilePath) {
```

Comment:
I do not believe this is necessary, the `baseDir` should be the same as what was passed into the function originally.